src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	
```

Comment:
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.